packages/caver-transaction/src/transactionTypes/abstractTransaction.js

```diff
@@ -306,6 +306,8 @@ class AbstractTransaction {
      */
     getRLPEncodingForSignature() {
         this.validateOptionalValues()
+        if (this.chainId === undefined)
```

Comment:
If this value has integer, i think ti should be better below code
if(this.chainID === undefined || _isNumber(hex)) or if(this.chainID === undefined || isHex(hex))